### PR TITLE
[13.0][ADD] partner_iterative_archive

### PR DESCRIPTION
**Pull request by astirpe**
This module automatically archives contacts when their parent is archived.

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by misern2 — APPROVED** [review]

File: partner_iterative_archive/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019-2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Iterative Archive",
+    "summary": "Archive all contacts when parent is archived",
```
Comment (bodedra) [inline partner_iterative_archive/__manifest__.py:6]:
```suggestion
    "summary": "Archive child contact when parent is archived",
```

File: partner_iterative_archive/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module automatically archives contacts when their parent is archived.
```
Comment (bodedra) [inline partner_iterative_archive/readme/DESCRIPTION.rst:1]:
```suggestion
This module automatically archives child contact when their parent is archived.
```

**Review by bodedra — APPROVED** [review]
LG :+1: 

Added minor suggestion.

**Pull request merged**